.circleci/config.yml

```diff
@@ -36,6 +36,15 @@ jobs:
           paths:
             - node_modules
           key: v1-dependencies-{{ checksum "package-lock.json" }}
+      - run:
+          name: "Run build"
+          command: 'npm run build'
+      - save_cache:
+          paths:
+            - common-dist
+            - dist
+          key: asset-cache-{{ .Environment.CIRCLE_SHA1 }}
```

Comment:
store `common-dist` and `dist` so other jobs in the workflow can just load it from the cache